exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,47 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `compress` (optional): Option to enable compression (default true)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
+- `log_format` (optional) (logs only): Format to post logs into Sumo. (default `json`)
+  - `text` - Logs will appear in Sumo Logic in text format
+  - `json` - Logs will appear in Sumo Logic in json format.
+- `metric_format` (optional) (metrics only): The format of metrics you will be sending, either graphite or carbon2 or prometheus (Default is carbon2).
```

Comment:
Just keep in mind both logs and metrics can be served by the same endpoint. 
Backend distinguishes metrics from logs checking at content type `Content-Type: application/vnd.sumologic.<metrics_type>`

Reply:
👍  I'm aware of that, I added `metrics/logs only` info because of otelcol architecture